### DISCUSSION TITLE
Laravel Nova support available?

### DISCUSSION THREAD

**Discussion by goellner** [staudenmeir/eloquent-has-many-deep#183] [Q&A] [answered]
Hello, 

just wanted to ask if there is a package to use HasManyDeep in Laravel Nova. 

kind regards,
Stefan

**Comment by staudenmeir** [comment]
Hi @goellner,
I can't really help you with this, I've never used Nova.

> if there is a package to use HasManyDeep in Laravel Nova

Do you need an additional package for Nova?

**Reply by goellner** [reply]
My mistake, it works when I set it up as a regular HasMany in Nova. All good, thank you

**Comment by goellner** [comment, marked as answer]
Works with a regular HasMany field in Laravel Nova
